src/modules/app/actions/init-augur.js

```diff
@@ -35,51 +42,31 @@ function pollForAccount(dispatch, getState) {
         }
       }
     })
-  }, POLL_INTERVAL_DURATION)
+  }, ACCOUNTS_POLL_INTERVAL_DURATION)
 }
 
 function pollForNetwork(dispatch, getState) {
-  let networkId
-
   setInterval(() => {
-    const { modal, env } = getState()
-    const expectedNetwork = env['network-id']
-
-    if (parseInt(AugurJS.augur.rpc.getNetworkID(), 10) !== networkId) {
-      networkId = parseInt(AugurJS.augur.rpc.getNetworkID(), 10)
-
-      if (networkId !== expectedNetwork && isEmpty(modal)) {
-        const getNetworkName = () => {
-          switch (expectedNetwork) {
-            case 1:
-              return 'Mainnet'
-            case 4:
-              return 'Rinkeby'
-            case 12346:
-              return 'Private'
-            default:
-              return expectedNetwork
-          }
-        }
-
+    const { modal } = getState()
+    verifyMatchingNetworkIds((err, isMatchingNetworkIds) => {
+      if (err) return console.error('pollForNetwork failed', err)
+      if (isMatchingNetworkIds === false && isEmpty(modal)) {
+        const expectedNetworkId = AugurJS.augur.rpc.getNetworkID()
```

Comment:
This seems to be the network metamask is using. Should it be the network of the ether connected ether node?

<img width="933" alt="screen shot 2018-03-07 at 11 23 41 pm" src="https://user-images.githubusercontent.com/328965/37138415-d6cf5f08-225e-11e8-8644-d2f8e63d896f.png">